### DISCUSSION TITLE
[HELP] The x3d file generated by Vedo can not be showed up in html.

### DISCUSSION THREAD

**Discussion by ZiguoAtGitHub** [marcomusy/vedo#746] [Q&A] [answered]
I just copy the example (light.py) and Press Shift+F in the Vedo window. 
The scene.html and scene.x3d can be generated automatically by Vedo. However I open the scene.html file and no model can be seen with my newest verison of  Microsoft Edge.
Is something wrong with my doing?
Thanks!

**Comment by marcomusy** [comment, marked as answer]
If you try to open it locally your browser is probably blocking it (press F12 to inspect the console output for error messages).
Using chrome you can enable it with:
`google-chrome --enable-webgl --allow-file-access-from-files`

You need to investigate how to achieve the same in Microsoft Edge.

**Reply by ZiguoAtGitHub** [reply]
Thank you Marcomusy.
It is strange that I can open vedo home page with my Edge browser perfectly and the scene in the middle of the page can be shown and operated with mouse as well.
I uploaded the scene.html and sene.x3d files to my own website but the problem is still there. Maybe you can take a look by clicking the website http://www.xfigure.cn/xvedo.html to find what happened.
Thank you again.

**Reply by marcomusy** [reply]
if you press F12 (in your browser) you will see that the `scene.x3d` file is not present (or accessible)!

http://www.xfigure.cn/scene.x3d

**Reply by ZiguoAtGitHub** [reply]
Hi Marco, @marcomusy 
I modify the scene.html file by following your hints. It works!!
Thank you very much!
You can have a look at http://www.xfigure.cn/xvedo.html if you are interested.
😃

**Reply by marcomusy** [reply]
Nice!  thanks for sharing!
